src/js/components/InfiniteScroll/InfiniteScroll.js

```diff
@@ -151,12 +151,12 @@ const InfiniteScroll = ({
   }, [items.length, lastPage, onMore, pendingLength, renderPageBounds, step]);
 
   useEffect(() => {
-    if (lastPage === 0 && pendingLength !== 0) {
+    if (items.length && lastPage === 0 && pendingLength !== 0) {
```

Comment:
Think we lost the `=== 0`

```suggestion
    if (items.length === 0 && lastPage === 0 && pendingLength !== 0) {
```

Reply:
Oops. My bad. Will fix it in a sec!